examples/vue/src/BlockNoteView.vue

```diff
@@ -0,0 +1,51 @@
+<script lang="ts" setup>
+import { onMounted, ref } from "vue"
+import { BlockNoteEditor } from "@blocknote/core"
+import type { BlockNoteEditorOptions } from "@blocknote/core"
+import { EditorContent } from "@tiptap/vue-3"
+import { blockSideMenuFactory } from "@/ui/blockSideMenuFactory"
+
+const props = defineProps<{
+  modelValue: string
+  options?: BlockNoteEditorOptions
+}>()
+
+const emit = defineEmits(['update:modelValue'])
+
+const editor = ref()
+onMounted(async () => {
+  // Convert md to html
+  const Editor = new BlockNoteEditor({})
+  const blocks = await Editor.markdownToBlocks(props.modelValue)
+  const content = await Editor.blocksToHTML(blocks)
```

Comment:
@YousefED This is ugly. How to pass initial content ?

Reply:
We'll add a new method + demo for this next week! Best to always work with `JSON.stringify(blocks)` btw, instead of storing markdown

See https://github.com/TypeCellOS/BlockNote/issues/139